src/components/Nav.js

```diff
@@ -1,23 +1,75 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
-import '../styles/Nav.css';
 
 const Nav = () => {
   return (
-    <nav>
-      <ul className="flexed-list">
-        <li className="list">
-          <NavLink to="/list" activeClassName="active">
-            List
-          </NavLink>
-        </li>
-        <li className="list">
-          <NavLink to="/addItem" activeClassName="active">
-            Add Item
-          </NavLink>
-        </li>
-      </ul>
-    </nav>
+    <div>
+      <div className="md:hidden">
+        <nav className="h-auto fixed bottom-0 right-0 left-0">
+          <ul className="flex justify-center content-center fixed bottom-0 right-0 left-0 m-auto bg-green-300 rounded-t-3xl shadow-top">
+            <li className="list-none mr-8">
+              <NavLink to="/list" activeClassName="font-bold">
+                <svg
+                  xmlns="http://www.w3.org/2000/svg"
+                  className="w-20 mt-5 mb-5 fill-current text-green-500 shadow-xl rounded-full"
+                  viewBox="0 0 375 375"
+                  version="1.2"
+                >
+                  <defs>
```

Comment:
For reading reasons it would be better to keep the svgs in an img folder and import them as a component!

Reply:
Yes Thank you! I meant to move these!